types/test/tsconfig.json

```diff
@@ -3,7 +3,9 @@
 		"target": "es5",
 		"module": "es2015",
 		"strict": true,
-		"noEmit": true
+		"noEmit": true,
+		"skipLibCheck": true,
```

Comment:
Why are these 2 options now needed?

Reply:
There were some TypeScript issues in Chart.js interfaces(https://github.com/chartjs/Chart.js/issues/7838)
Seems fixed, so will test reverting this.

Reply:
Removed, not needed anymore.